udf/udf_active_addons_mode_last.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TEMP FUNCTION
+  udf_active_addons_mode_last(nested_list ANY TYPE) AS (STRUCT(ARRAY(
+      SELECT
+        STRUCT(udf_json_mode_last(ARRAY_AGG(element)) AS element)
+      FROM
+        UNNEST(nested_list),
+        UNNEST(list)
```

Comment:
i'll add a comment about the input schema (it's left out here because it's big, and only part of it is actually referenced in the function). as for the input field name, i'll make it more verbose.